src/app/src/util/constants.jsx

```diff
@@ -32,7 +32,7 @@ export const InfoPaths = {
     researchers: 'researchers',
     serviceProviders: 'service-providers',
     sectors: 'sectors',
-    technology: 'technology',
+    technology: 'technology-overview',
```

Comment:
When I follow this link (/technology-overview), it leads to a 404 page. Oddly, the info site's header currently leads to /technology, which works. I suspect they've changed it since the issue was made earlier.